<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAXSgui</class>
 <widget class="QMainWindow" name="SAXSgui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>630</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>925</width>
    <height>630</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>UCSFsaxs v0.6</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Console">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>821</width>
      <height>541</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="fourierspace_tab">
     <attribute name="title">
      <string>Fourier Space (Scattering)</string>
     </attribute>
     <widget class="GraphicsLayoutWidget" name="fourierView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>511</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkLogY">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Log-scale Y-axis</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkLogX">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>40</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Log-scale X-axis</string>
      </property>
     </widget>
     <widget class="GraphicsLayoutWidget" name="beamview">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>300</y>
        <width>261</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>280</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Beam Profile</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="nskip_spinbox">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>80</y>
        <width>57</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>80</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Initial Points to skip</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="nskip2_spinbox">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>110</y>
        <width>57</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>110</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>End Points to skip</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>160</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>q_min : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>180</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>q_max : </string>
      </property>
     </widget>
     <widget class="QLabel" name="qmin_label">
      <property name="geometry">
       <rect>
        <x>605</x>
        <y>160</y>
        <width>206</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="qmax_label">
      <property name="geometry">
       <rect>
        <x>605</x>
        <y>180</y>
        <width>206</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="guinierkratky_tab">
     <attribute name="title">
      <string>Guinier, Kratky Plots</string>
     </attribute>
     <widget class="GraphicsLayoutWidget" name="primaryanalysis_view">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>781</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>341</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Automated Guinier Analysis</string>
      </property>
      <widget class="QDoubleSpinBox" name="qRgmin">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>62</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>62</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>qRg_min</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>62</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>qRg_max</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="qRgmax">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>62</width>
         <height>24</height>
        </rect>
       </property>
       <property name="maximum">
        <double>3.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.300000000000000</double>
       </property>
      </widget>
      <widget class="QLineEdit" name="guinierminpts">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Min. points</string>
       </property>
      </widget>
      <widget class="QPushButton" name="autoguinier_button">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>55</y>
         <width>91</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #F1BF29; font-size: 12px; color: green;</string>
       </property>
       <property name="text">
        <string>Do it (Auto)!</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="runprimaryanalysis">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>435</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #F1BF29; font-size: 12px; color: blue;</string>
      </property>
      <property name="text">
       <string>Do it (Manual)!</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="lb_guinier_spinbox">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>435</y>
        <width>57</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="ub_guinier_spinbox">
      <property name="geometry">
       <rect>
        <x>495</x>
        <y>435</y>
        <width>57</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>475</x>
        <y>440</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>to</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>440</y>
        <width>26</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>375</x>
        <y>470</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>current qRg : </string>
      </property>
     </widget>
     <widget class="QLabel" name="qRg_label">
      <property name="geometry">
       <rect>
        <x>465</x>
        <y>470</y>
        <width>126</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>375</x>
        <y>490</y>
        <width>36</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rg : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>490</y>
        <width>36</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>I(0):</string>
      </property>
     </widget>
     <widget class="QLabel" name="guinierRg_label">
      <property name="geometry">
       <rect>
        <x>405</x>
        <y>490</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
     <widget class="QLabel" name="I0_label">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>490</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="realspace_tab">
     <attribute name="title">
      <string>Real Space</string>
     </attribute>
     <widget class="GraphicsLayoutWidget" name="realspaceView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>531</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>0</y>
        <width>261</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>IFT parameters</string>
      </property>
      <widget class="QLineEdit" name="Nr_input">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>166</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Log(alpha) - Smoothness</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Dmax</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string># of points in P(r)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="logalpha_input">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="dmax_input">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>210</y>
        <width>151</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Data Parameters</string>
      </property>
      <widget class="QCheckBox" name="varianceweighted">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>σ^2 weighted</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="smeared">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>55</y>
         <width>87</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Smeared</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="solvepr_button">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>440</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0085ff; color: #ffffff;</string>
      </property>
      <property name="text">
       <string>Solve P(r)!</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="iftresultout">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>400</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="saveiftresult">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>370</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Save File? Enter file name below:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="bayesianift_tab">
     <attribute name="title">
      <string>BayesianIFT</string>
     </attribute>
     <widget class="matplotlibWidget" name="biftgridView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>401</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>0</y>
        <width>381</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Grid Search Parameters</string>
      </property>
      <widget class="QLineEdit" name="Nalpha">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Ndmax">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>No. of alpha</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>No. of Dmax</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="minlogalpha">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>62</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>log(alpha) Range</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="maxlogalpha">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>62</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>100</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Dmax Range</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="mindmax">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>120</y>
         <width>62</width>
         <height>24</height>
        </rect>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="maxdmax">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>120</y>
         <width>62</width>
         <height>24</height>
        </rect>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="calcbiftgrid">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>180</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #6495ed; color: #ffffff;</string>
      </property>
      <property name="text">
       <string>Calculate Grid</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>120</y>
        <width>10</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>120</y>
        <width>10</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Ncalcerror_input">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>310</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>310</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No. of error calculations (&gt;100)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refinebift">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>350</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#87cc83; color: #ffffff;</string>
      </property>
      <property name="text">
       <string>Refine Solution (Simplex)</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>270</y>
        <width>391</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as_GNOM_file"/>
    <addaction name="actionLoad_Beam_Profile"/>
    <addaction name="actionClose"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open SAXS data file</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit Program</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionGuinier_Plot">
   <property name="text">
    <string>Guinier Plot</string>
   </property>
  </action>
  <action name="actionKratky_Plot">
   <property name="text">
    <string>Kratky Plot</string>
   </property>
  </action>
  <action name="actionSAXSMoW">
   <property name="text">
    <string>SAXSMoW</string>
   </property>
  </action>
  <action name="actionFoXS_fit">
   <property name="text">
    <string>FoXS fit ...</string>
   </property>
  </action>
  <action name="actionLoad_Beam_Profile">
   <property name="text">
    <string>Load Beam Profile</string>
   </property>
  </action>
  <action name="actionSave_as_GNOM_file">
   <property name="text">
    <string>Save as GNOM file ...</string>
   </property>
  </action>
  <action name="actionSAXSMoW_2">
   <property name="text">
    <string>SAXSMoW</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>matplotlibWidget</class>
   <extends>QGraphicsView</extends>
   <header>mplWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
